content/millennium/r4/encounters/encounter.md

```diff
@@ -9,4 +9,56 @@ title: Encounter | R4 API
 
 ## Overview
 
-Placeholder
+The Encounter resource provides admissions or visits during which health care services were provided to a patient. An encounter has a class to distinguish between different health care settings such as inpatient, outpatient, emergency, etc. A patient may have one medical record number with multiple encounter numbers per facility or organization. There is substantial variance between organizations in the definition of an encounter and what events are aggregated together to constitute an encounter.
+
+## Patch
+
+Patch an existing encounter.
+
+    PATCH /Encounter/:id
+
+_Implementation Notes_
+
+* Only operations on the paths listed below are supported.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', 'Accept': 'application/fhir+json',
+                   'Content-Type': 'application/json-patch+json', 'If-Match': 'W/"&lt;Current version of the Encounter resource>"'} %>
+
+### Patch Operations
+
+<%= patch_definition_table(:encounter_patch, :patch, :r4) %>
+
+### Example
+
+#### Request
+
+    PATCH https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Encounter/1621910
+
+#### Body
+
+<%= json(:r4_encounter_patch) %>
+
+#### Response
+
+<%= headers status: 200 %>
+<pre class="terminal">
+    Cache-Control: no-cache
```

Comment:
since we're already using the html pre tag, indenting 4 spaces ends up just putting 4 spaces in front of all the lines

we can remove the spaces

Reply:
removed. https://github.com/cerner/fhir.cerner.com/commit/b3be566e03125de4bdc242726080efeaaef55e65#diff-6590262e197180cc09f6ad70a2a01a5bR51